internal/cfgplugins/staticroute.go

```diff
@@ -48,12 +58,64 @@ func NewStaticRouteCfg(batch *gnmi.SetBatch, cfg *StaticRouteCfg, d *ondatra.DUT
 	}
 	s := c.GetOrCreateStatic(cfg.Prefix)
 	for k, v := range cfg.NextHops {
+		if cfg.NexthopGroup {
```

Comment:
Shouldn't  the next hop group creation be done before looping through the next-hops?  ie: there's just one NHG which contains 1 or more next-hops